tools/contrib/fsurdat_modifier.py

```diff
@@ -0,0 +1,356 @@
+#! /usr/bin/env python
+"""
+|------------------------------------------------------------------|
+|---------------------  Instructions  -----------------------------|
+|------------------------------------------------------------------|
+
+Instructions for running on Cheyenne/Casper:
+
+load the following into your local environment
+    module load python
+    ncar_pylib
+
+-------------------------------------------------------------------
+To see available options:
+    ./fsurdat_modifier.py --help
+-------------------------------------------------------------------
+
+This script reads a surface dataset (fsurdat file) and outputs a
+modified copy of the same file.
+
+-------------------------------------------------------------------
+To run the script:
+    ./fsurdat_modifier.py
+ 
+To remove NPL (ncar_pylib) from your environment on Cheyenne/Casper:
+    deactivate
+-------------------------------------------------------------------
+
+"""
+
+#  Import libraries
+from __future__ import print_function
```

Comment:
- [ ] If you change the shebang line to python3 (as you should), then this isn't necessary